src/dss/ClipAbstract.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+/// ClipAbstract.sol -- Clip Interface
+
+// Copyright (C) 2021 Maker Ecosystem Growth Holdings, INC.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.6.12;
+
+interface ClipAbstract {
+    function wards(address) external view returns (uint256);
+    function rely(address) external;
+    function deny(address) external;
+    function ilk() external view returns (bytes32);
+    function vat() external view returns (address);
+    function dog() external view returns (address);
+    function vow() external view returns (address);
+    function spotter() external view returns (address);
+    function calc() external view returns (address);
+    function buf() external view returns (uint256);
+    function tail() external view returns (uint256);
+    function cusp() external view returns (uint256);
+    function chip() external view returns (uint64);
+    function tip() external view returns (uint192);
+    function chost() external view returns (uint256);
+    function kicks() external view returns (uint256);
+    function active() external view returns (uint256[] memory);
```

Comment:
Solidity exposes an indexed accessor by default (hence why we need `list()` below), so this should read:
```
function active(uint256) external view returns (uint256);
```